examples/compareLambdaTypes.py

```diff
@@ -257,18 +260,18 @@ def applyVisitor(self, invert=False, xloc=397, yloc=580):
             kImageOut2.writeFits('k4.fits')
             diffIm2.writeFits('d4')
 
-
-        raw_input('Next: ')
+        input('Next: ')
 
     def testFunctor(self):
         for fp in self.footprints:
             # note this returns the kernel images
-            self.applyVisitor(invert=False, 
-                              xloc= int(0.5 * ( fp.getBBox().getMinX() + fp.getBBox().getMaxX() )),
-                              yloc= int(0.5 * ( fp.getBBox().getMinY() + fp.getBBox().getMaxY() )))
-       
+            self.applyVisitor(invert=False,
+                              xloc=int(0.5 * (fp.getBBox().getMinX() + fp.getBBox().getMaxX())),
+                              yloc=int(0.5 * (fp.getBBox().getMinY() + fp.getBBox().getMaxY())))
+
 #####
-        
+
+
 def suite():
```

Comment:
These `compare*` files appear to have been written by Andy Becker in the 2009-2011 time frame. It may be hard to get them all to run, and we'd have to see whether they're actually asserting anything, or if they used the TestSuite stuff as helpers for running the code.